hadoop-hdds/server-scm/src/main/resources/webapps/scm/scm-overview.html

```diff
@@ -235,6 +237,34 @@ <h2>Meta-Data Volume Information</h2>
     </tbody>
 </table>
 
+<h2 ng-show="$ctrl.overview.jmx.RatisRoles.length > 1">SCM Roles (HA)</h2>
```

Comment:
From my previous comment regarding re-adding the line

`<h4 ng-show="$ctrl.overview.jmx.RatisRoles.length == 1 && $ctrl.overview.jmx.RatisRoles[0].length == 1">{{$ctrl.overview.jmx.RatisRoles[0][0]}}</h4>`

We can also remove the conditional since to still show the header even during exception.

```suggestion
<h2>SCM Roles (HA)</h2>
```

We can basically copy the implementation for OM HA UI.